00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -81,6 +80,11 @@
     "\n",
     "Depending on your environment, you might need to customize the setup when creating your Bedrock service client. To help with this, we've provided a `get_bedrock_client()` utility method that supports passing in different options. You can find the implementation in [../utils/bedrock.py](../utils/bedrock.py)\n",
     "\n",
+    "#### Use different clients\n",
+    "The boto3 provides different clients for Amazon Bedrock to perform different actions for data plane and control plane. The actions for [`InvokeModel`](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InvokeModel.html) and [`InvokeModelWithResponseStream`](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InvokeModelWithResponseStream.html) are supported by Amazon Bedrock Runtime where as other control plane actions are handled via [Amazon Bedrock client](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_Operations_Amazon_Bedrock.html).\n",
+    "\n",
+    "To get different client, you can pass in the `mode` parameter, with the values `data-plane` or `control-plane` to get bedrock-runtime or bedrock clients respectively.\n",
```

Comment:
please remove control-plane and data-plane references.

Reply:
Updated